functions/helloworld/hello-pub-sub/src/main/java/functions/eventpojos/PubSubMessage.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package functions.eventpojos;
```

Comment:
I'm Ok with this, but to truely be a POJO, you should either use AutoValue, or implement `equals()`, `toString()`, and `hashCode()` methods.

Reply:
Ditto to above - see b/153731328.